packages/cfd/src/types/market-type.types.ts

```diff
@@ -0,0 +1,3 @@
+export type TMarketTypeSynthetic = 'synthetic';
```

Comment:
This file is made for explicitly the type and in future we can use this folder to add more types.
When we are going ot remove `Components/props.types` and `Container/props.types`.